app/views/review_mapping/_searchbox.html.erb

```diff
@@ -55,6 +55,22 @@
       <br>
     </div>
 
+    <div id="MetricsForm">
```

Comment:
Another place where the code will have to be modified each time a metric is added.